### ISSUE TITLE
sphinx-lint does not detect missed backtick pair

### ISSUE THREAD

**Issue by zaufi** [sphinx-contrib/sphinx-lint#128]
I (accidentally) found a few typos in my docs that look like this:
```rts
``COMPONENT` here is the description of this parameter...
```
Due to the missed 2nd backtick, the rest of the line was rendered italicized. It would be nice if `sphinx-lint` could detect these cases.

P.S. Thanks for the nice linter!

**Comment by hugovk** [comment]
What version are you using? I get a warning with the latest 1.0.0:

```console
❯ sphinx-lint --version
sphinx-lint 1.0.0

❯ cat 1.rst
``COMPONENT` here is the description of this parameter...

❯ sphinx-lint 1.rst
1.rst:1: found an unbalanced inline literal markup. (unbalanced-inline-literals-delimiters)
```

> P.S. Thanks for the nice linter!

You're welcome :)

**Comment by zaufi** [comment]
Sorry, you are right! It can find this case. My problem is that RST docs are in CMake files (as comments) and aren't checked by `sphinx-lint` (I'm using the `pre-commit` hook).
